source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,591 @@
+module Fluent
+
+  class OutChangeTrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
```

Comment:
Can you provide more explanation here? Can you share a link to a conf file which uses this tag? Also, tag and plugin name do not have to be related; data with a certain tag can go through any number of plugins with any number of names.

We do not have a strongly enforced convention, but if you look at the core output plugins, they use the convention I mentioned (see [out_oms_blob.rb](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/source/code/plugins/out_oms_blob.rb)).